.travis.yml

```diff
@@ -3,6 +3,3 @@ node_js:
   - 0.12
   - 4
   - 5
-
-before_install:
-  - npm install npm@latest -g
```

Comment:
The travis build for node v0.12 was failing until I removed this. There was an error message saying that the latest version of npm doesn't support node v0.12